tests/integration/redis_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
 // +build integration
```

Comment:
Can we remove the older format?

Reply:
Done in [#6ad7c35](https://github.com/newrelic/nri-redis/pull/203/commits/6ad7c3587fd18e72d9bdc3eac21e0e3d9b4c1d64)